sdk/identity/azure-identity/src/test/java/com/azure/identity/ManagedIdentityCredentialTest.java

```diff
@@ -21,6 +21,8 @@
 import java.time.ZoneOffset;
 import java.util.UUID;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
```

Comment:
Any reason hamcrest was pulled in as a testing dependency instead of using the JUnit 5 assertions?

Reply:
Chatted with @alzimmermsft offline. We're using junit4 in this library still, so we'll go with this until we can upgrade.

Reply:
Currently, Identity uses JUnit 4.
So, will have to refactor when we change the test suite to Junit 5.